src/widgets/helper/ChannelView.cpp

```diff
@@ -1980,6 +1972,15 @@ void ChannelView::mouseReleaseEvent(QMouseEvent *event)
             {
                 return;
             }
+
+            // Triple-clicking a message selects the whole message
+            if (this->clickTimer_->isActive() &&
+                (fabsf(distanceBetweenPoints(this->lastDClickPosition_,
+                                             event->screenPos())) < 10.f))
+            {
+                this->selectWholeMessage(layout.get(), messageIndex);
```

Comment:
This can segfault if you double-click on the bottom of the last message, then move the cursor below the last message (distance must be less than 10px). Probably needs a check if `foundElement` is true.

Reply:
fixed in 662c7ab88a48254d65eb47fad1c6a7edf91bf767, thanks!